desktop/src/Desktop/Frontend.hs

```diff
@@ -1,14 +1,14 @@
-{-# LANGUAGE ConstraintKinds #-}
```

Comment:
It looked like we used stylish haskell in a different file that I changed earlier today, but this one did not have the autoformat previously applied. Should I have this autoformat turned on or off working on chainweaver?

Reply:
off please; the diffs are bad otherwise

Reply:
Fixed, as well as adding in https://github.com/kadena-io/chainweaver/pull/223 to get consistency around the place.